vm-virtio/src/vhost_user/handler.rs

```diff
@@ -94,7 +96,7 @@ impl<S: VhostUserMasterReqHandler> VhostUserEpollHandler<S> {
             None
         };
 
-        let mut events = vec![epoll::Event::new(epoll::Events::empty(), 0); kill_evt_index + 1];
+        let mut events = vec![epoll::Event::new(epoll::Events::empty(), 0); index + 1];
```

Comment:
Go with the old code and do this instead:

`let mut events = vec![epoll::Event::new(epoll::Events::empty(), 0); slave_evt_index.unwrap_or(kill_evt_index) + 1];`

Reply:
Yes, we could do that, but it doesn't seem to be good for expansion if there are new events added in future, and the logic in match branch also needs to be update accordingly.